modules/task/backend.go

```diff
@@ -30,6 +30,12 @@ import (
 
 var gobEncoding = &GobEncoding{}
 
+const (
+	_initialized = iota
```

Comment:
i think the convention for the first one is typically `unknown` or something like that, and the three valid states are listed below. I might be wrong though